### PR TITLE
(Deposit/Withdraw) Add Nomic Deposit Address Flow and Withdraw

### PR DESCRIPTION
**Pull request by JoseRFelix**
#### These changes are behind a feature flag. 

Add the Deposit Address UI and update the Nomic Bridge provider to support deposits. 

### Linear Task

https://linear.app/osmosis/issue/FE-1119/add-nomic-bridge-provider

## Testing and Verifying

- [ ] Deposit BTC displays the new deposit address screen

### CODE REVIEW

**Review by jonator — CHANGES_REQUESTED** [review]
I wanted to test a disabled asset.

It works, but it appears we're now showing network selectors at the top in the dedicated external URLs screen:
Screenshot 2024-10-07 at 7 03 18 PM

File: packages/bridge/src/nomic/index.ts [hunk 1]
```diff
@@ -16,6 +23,94 @@ export class NomicBridgeProvider implements BridgeProvider {
...
+
+    if (!transferMethod) {
+      throw new Error("IBC transfer method not found for Nomic Bitcoin asset.");
+    }
+
+    if (fromChain.chainId !== "bitcoin") {
+      throw new Error("Only Bitcoin is supported as a source chain.");
+    }
+
+    const depositInfo = await generateDepositAddressIbc({
```
[the remark this answers is not in the document]
Comment (jonator) [reply packages/bridge/src/nomic/index.ts:61]: But it's on server so can disregard

**Comment by JoseRFelix** [comment]
>It works, but it appears we're now showing network selectors at the top in the dedicated external URLs screen:

Yes, this is intentional. Our goal is to avoid blocking users on a specific chain or asset when no providers are available, especially if there are other supported chains available

File: packages/bridge/src/nomic/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoseRFelix) [reply packages/bridge/src/nomic/index.ts:61]: Yeah, I didn't do an async import since it will not be imported on the client

**Pull request merged**